cleantalk.antispam/options.php

```diff
@@ -710,6 +734,20 @@ function ctDisableInputLine(ct_input_line){
                     value="1" />
         </td>
     </tr>
+    <tr>
+        <td width="50%" valign="top">
+            <label for="use_custom_server"><?php echo GetMessage( 'CLEANTALK_USE_CUSTOM_SERVER' );?>:</td>
+        <td  valign="top">
+            <input
+                    type="text"
+                    name="use_custom_server"
+                    id="use_custom_server"
+                    value="<?php echo $current_options['use_custom_server'] == '1' ? 'cleantalk.ru' : ''; ?>" />
```

Comment:
Not allowed to hardcode 'ru' (and other) zone. Get this by option.

Reply:
Thanks, fixed https://github.com/CleanTalk/bitrix-antispam/pull/30/commits/157052b77157dbced40379f1ae89a604639c978c